index.html

```diff
@@ -44,7 +44,7 @@
   </head>
   <body>
 
-    <h1>No</h1>
+    <h1>Yes</h1>
     <script>
       // beware of leading zeroes, they indicate octal.
       countup(2016,6,23)
```

Comment:
The counter has not been stopped. Based on the changes from @RobertThz in db0490895e4a262fcf29fd5bba555a7031f2c31b, it looks like this should be changed to `countup(2016,6,23, 2017,3,29)`